src/pages/AdminSettings/AdminMembers/MemberList/MemberTable.tsx

```diff
@@ -94,6 +93,28 @@ const createTable = ({ tableData, seatData, mutate }: CreateTableArgs) => {
   )
 }
 
+interface Params {
+  activated?: boolean
+  search?: string
+  isAdmin?: boolean
+}
+
+const useQueryMembersList = (params: Params) => {
```

Comment:
Im also kinda curious why we pulled the memo out of the query and into this new query in the component itself

Reply:
makes sense!